### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by mattd-kainos**

### CODE REVIEW

File: test-scripts/1.0.0/export-env-vars.opentest.example.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+export NHAIS_MESH_CYPHER_TO_MAILBOX="XX11=nhais_mailbox
+TES5=nhais_mailbox
+YY21=nhais_mailbox"
+
+# See adaptor's README.md for detailed description of MESH mailbox polling behaviour
+export NHAIS_SCAN_MAILBOX_DELAY_IN_SECONDS=300 # default: 300 (5 minutes)
+export NHAIS_SCAN_MAILBOX_INTERVAL_IN_MILLISECONDS=60000 # default: 60000 (1 minute)
+
+export NHAIS_MESH_SHARED_KEY=
+export NHAIS_MESH_HOST=https://msg.opentest.hscic.gov.uk/messageexchange/
```
Comment (grzegorz-robak) [inline test-scripts/1.0.0/export-env-vars.opentest.example.sh:17]: address should not include `/messageexchange/`

Comment (grzegorz-robak) [reply test-scripts/1.0.0/export-env-vars.opentest.example.sh:17]: or it should... I need to recheck that

Comment (grzegorz-robak) [reply test-scripts/1.0.0/export-env-vars.opentest.example.sh:17]: Ok, it should have it :)

File: mesh/all_inbound.dat [hunk 2]
```diff
@@ -0,0 +1,98 @@
+UNB+UNOA:2+XX11+TES5+920125:1235+00000003'
```
Comment (mattd-kainos) [inline mesh/all_inbound.dat]: remove before merge?

**Review by petervdm — APPROVED** [review]

**Pull request merged**